frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs

```diff
@@ -120,132 +157,315 @@
-     :on-click #(set-wrap :nowrap)
-     :style {:padding 0}}
-    [:span.no-wrap i/minus]]
-   [:button.wrap.tooltip.tooltip-bottom
-    {:class  (dom/classnames :active  (= wrap-type :wrap))
-     :alt    "Wrap"
-     :on-click #(set-wrap :wrap)}
-    i/auto-wrap]])
+  [{:keys [wrap-type set-wrap toggle-wrap] :as props}]
+  (let [new-css-system (mf/use-ctx ctx/new-css-system)]
+    (if new-css-system
+      [:button {:class (stl/css-case :wrap-button true
+                                     :selected (= wrap-type :wrap))
+                :title (if (= :wrap wrap-type)
+                         "No wrap"
+                         "Wrap")
+                :on-click toggle-wrap}
+       i/wrap-refactor]
+
+      [:*
+       [:button.tooltip.tooltip-bottom
+        {:class  (dom/classnames :active  (= wrap-type :nowrap))
+         :alt    "No wrap"
+         :on-click #(set-wrap :nowrap)
+         :style {:padding 0}}
+        [:span.no-wrap i/minus]]
+       [:button.wrap.tooltip.tooltip-bottom
+        {:class  (dom/classnames :active  (= wrap-type :wrap))
+         :alt    "Wrap"
+         :on-click #(set-wrap :wrap)}
+        i/auto-wrap]])))
 
 (mf/defc align-row
   [{:keys [is-col? align-items set-align] :as props}]
-
-  [:div.align-items-style
-   (for [align [:start :center :end #_:stretch #_:baseline]]
-     [:button.align-start.tooltip
-      {:class    (dom/classnames :active  (= align-items align)
-                                 :tooltip-bottom-left (not= align :start)
-                                 :tooltip-bottom (= align :start))
-       :alt      (dm/str "Align items " (d/name align))
-       :on-click #(set-align align)
-       :key      (dm/str "align-items" (d/name align))}
-      (get-layout-flex-icon :align-items align is-col?)])])
+  (let [new-css-system (mf/use-ctx ctx/new-css-system)]
+    (if new-css-system
+      [:& radio-buttons {:selected (d/name align-items)
+                         :on-change set-align
+                         :name "flex-align-items"}
+       [:& radio-button {:value "start"
+                         :icon  (get-layout-flex-icon-refactor :align-items :start is-col?)
+                         :title "Align items start"
+                         :id     "align-items-start"}]
+       [:& radio-button {:value "center"
+                         :icon  (get-layout-flex-icon-refactor :align-items :center is-col?)
+                         :title "Align items center"
+                         :id    "align-items-center"}]
+       [:& radio-button {:value "end"
+                         :icon  (get-layout-flex-icon-refactor :align-items :end is-col?)
+                         :title "Align items end"
+                         :id    "align-items-end"}]]
+
+      [:div.align-items-style
+        [:button.align-start.tooltip.tooltip-bottom
+        {:class    (dom/classnames :active  (= align-items :start))
+         :alt      "Align items start"
+         :on-click #(set-align :start)}
```

Comment:
Please don't pass a newly created callbacks (not memoized ones), prefer use the `:data-align "start"` approach